superset/views/geocoding.py

```diff
@@ -230,200 +223,174 @@ def geocode(self) -> Response:
         self.stats_logger.incr("succesful_geocoding")
         return json_success('"OK"')
 
-    def _get_from_clause(self, table):
+    def _create_column_list(self, request_data: dict) -> list:
+        columns = []
+        street_column = request_data.get("streetColumn")
+        city_column = request_data.get("cityColumn")
+        country_column = request_data.get("countryColumn")
+        building_number_column = request_data.get("buildingNumberColumn")
+        if street_column:
+            columns.append(street_column)
+        if city_column:
+            columns.append(city_column)
+        if country_column:
+            columns.append(country_column)
+        if building_number_column:
+            columns.append(building_number_column)
+        columns.append(request_data.get("latitudeColumnName", "lat"))
+        columns.append(request_data.get("longitudeColumnName", "lon"))
```

Comment:
this is already done in the method calling this one, but I'm guessing that's also part of the refactoring still going on?